### ISSUE TITLE
'Element action params did not validate'

### ISSUE THREAD

**Issue by johnniefp** [verbb/formie#1146]
### Describe the bug

The error 'Element action params did not validate' appears when you try to change the status of a submission from the main Submissions listing page.

### Steps to reproduce

1. Go to Forms > Submissions
2. Tick some entries
3. Change Set Status to see error

### Form settings

n/a

### Craft CMS version

4.2.8

### Plugin version

2.0.13

### Multi-site?

No

### Additional context

Happens on local, staging and production.
Note: I do have some custom statues.

**Comment by johnniefp** [comment]
Note: I do have a few custom statuses incase that matters here.

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.13",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.14](https://github.com/verbb/formie/releases/tag/2.0.14)
